### PR TITLE
Implement /etcdstatus and /etcdhealth endpoints

### PR DESCRIPTION
**Pull request by prashanthpai**
Closes issue #85 
Signed-off-by: Prashanth Pai <ppai@redhat.com>

### CODE REVIEW

File: commands/peers/commands.go [hunk 1]
```diff
@@ -27,6 +27,20 @@ func (c *Command) Routes() rest.Routes {
 			HandlerFunc: getPeersHandler,
 		},
 		rest.Route{
+			Name:        "EtcdHealthPeer",
+			Method:      "GET",
+			Pattern:     "/peers/{peerid}/etcdhealth",
```
Comment (prashanthpai) [inline commands/peers/commands.go]: @atinmu @kshlm I don't mind renaming `/etcdhealth` here to just `/status`

**Comment by prashanthpai** [comment]
Sample output:

**List peers**:

~~~
[ppai@gd2-1 ~]$ curl -s -X GET http://127.0.0.1:23007/v1/peers | python -m json.tool
[
    {
        "addresses": [
            "127.0.0.1:23008"
        ],
        "id": "b81671ed-09f5-45c3-90f3-5eff9e6abb1f",
        "memberID": 3964455086412988326,
        "name": "gd2-1"
    },
    {
        "addresses": [
            "127.0.0.1:22008"
        ],
        "id": "bf328d29-3afd-4b73-aa9e-8d85a16abfb5",
        "memberID": 12073872630648513194,
        "name": "gd2-1"
    }
]
~~~

**Get etcd health of a peer**:

~~~
[ppai@gd2-1 ~]$ curl -s -X GET http://127.0.0.1:23007/v1/peers/b81671ed-09f5-45c3-90f3-5eff9e6abb1f/etcdhealth | python -m json.tool
{
    "health": "true"
}
~~~

**Get detailed etcd status of a peer**:

~~~
[ppai@gd2-1 ~]$ curl -s -X GET http://127.0.0.1:23007/v1/peers/b81671ed-09f5-45c3-90f3-5eff9e6abb1f/etcdstatus | python -m json.tool
{
    "dbSize": 24576,
    "header": {
        "cluster_id": 9089655872076002276,
        "member_id": 3964455086412988326,
        "raft_term": 2,
        "revision": 1
    },
    "leader": 12073872630648513194,
    "raftIndex": 481,
    "raftTerm": 2,
    "version": "3.1.0-rc.1"
}

~~~

**Comment by kshlm** [comment]
What's the difference between health and status? Is health even required?

**Comment by prashanthpai** [comment]
@kshlm While a successful response from both `/etcdhealth` and `/etcdstatus` mean the etcd server is up, `/etcdhealth` represents if health of the etcd instance is good or not (eg. misconfigured cluster, other peers unreachable etc). `etcdstatus` will give a detailed response from the instance but not the health information.

File: commands/peers/commands.go (hunk 1 above)
Comment (prashanthpai) [inline commands/peers/commands.go]: On a second thought, in future, if etcd isn't run on all nodes, then probably we should keep this distinction between `/etcdhealth` and `/status` endpoints.

**Pull request merged**